www/_includes/footer_contents.html

```diff
@@ -63,8 +63,8 @@ <h1>Contribute</h1>
     </div>
 </div>
 <p class="copyright_text">
-    Copyright &copy; {{ 'now' | date: "%Y" }} The Apache Software Foundation, Licensed under the <a target="_blank" href="http://www.apache.org/licenses/LICENSE-2.0">Apache License, Version 2.0</a>.<br/>
-    Apache and the Apache feather logos are <a target="_blank" href="http://www.apache.org/foundation/marks/list/">trademarks</a> of The Apache Software Foundation.
+    Copyright &copy; {{ 'now' | date: "%Y" }} The Apache Software Foundation, Licensed under the <a target="_blank" href="https://www.apache.org/licenses/LICENSE-2.0">Apache License, Version 2.0</a>.<br/>
```

Comment:
**Just for Information:**

Generally when adding a `LICENSE` file to a repo or appending the license header to source code files, it must contain the `http` version of the URL. It is offical documented in this way on Apache's website, eventhough `https` url works..

I believe it is something to do with how the notices were approved leagl in 2004? 

I am not entirely sure about this.

But in terms of the copyright text on the bottom of the webpage, I couldnt find any documentation on Apache explaining what is allowed.

So, IMO, I believe this change is safe.